ACPs/77-reinventing-subnets/README.md

```diff
@@ -249,7 +249,7 @@ Currently, the P-Chain operates under a fixed fee mechanism for each transaction
 
 Each additional Subnet Validator on the P-Chain adds load to the network. The discussions for the dynamic multi-dimensional fee charge when a transaction is instantiated but there isn't a solution for charging Subnet Validators for the space they are using over the time they are validating on the network (which may be indefinitely). This is a common problem in blockchains and there have been many state rent proposals in the broader blockchain space to address it. This fee mechanism takes advantage of the fact that each Subnet Validator uses the same amount of state and charges each Subnet Validator the dynamic base fee for every discrete unit of time it is active.
 
-To charge each Subnet Validator, the notion of a `Balance` is introduced. The `Balance` of a Subnet Validator will be continuously charged in proportion to the amount of time they are a Subnet Validator. When this `Balance` reaches `0`, the Subnet Validator will be removed from the validator set. This `Balance` is first set in the transaction that added them to the Subnet Validator set. This `Balance` can be subsequently topped up at any time using the `IncreaseBalanceTx`.
+To charge each Subnet Validator, the notion of a `Balance` is introduced. The `Balance` of a Subnet Validator will be continuously charged in proportion to the amount of time they are a Subnet Validator. This `Balance` is first set in the transaction that added them to the Subnet Validator set. This `Balance` can be subsequently topped up at any time using the `IncreaseBalanceTx`. When this `Balance` reaches `0`, the Subnet Validator will be considered an "Inactive" Subnet Validator and will no longer participate in validating the Subnet. Inactive Subnet Validators can be revived at any time using the `IncreaseBalanceTx`. The continuous fee is assessed only during the time that a Subnet Validator is active to cover the cost of storing the associated validator properties (BLS key, weight, nonce) in memory. Once a Subnet Validator is considered inactive, the P-Chain will remove these properties from memory and only retain them on disk. All messages from that validator will be considered invalid until it is revived using the `IncreaseBalanceTx`. Subnets can reduce the amount of inactive weight by using the `SetSubnetValidatorWeightTx` with `Weight` of `0`.
```

Comment:
nit: to cover the cost of storing the associated validator properties (BLS key, weight, nonce) in memory and to track IP (or other services provided by primary network)